pages/inspector/inspector-issues-view.mdx

```diff
@@ -25,22 +26,91 @@ Each issue in the table represents an issue on a particular event, or a particul
-- Issue Type
-- Source
-- Event name
-- Property
-- Category
-- Tag
-- Release
+
+- **Issue Type**: The [Inspector Issue Type](/inspector/issue-types-in-inspector)
+- **Source**: The source where the issue was seen
+- **Event**: The event impacted by the issue
+- **Property**: The property impacted by the issue
+- **Category**: The category of the event impacted by the issue
+- **Tag**: The tag of the event impacted by the issue
+- **Release**: The product release version impacted by the issue. Currently there are three release filters available:
+  - **Latest release**: The latest product release version seen on each of your sources
+  - **Two latest releases**: The two latest product release versions seen on each of your sources. This option can be helpful if the latest release is not yet fully rolled out.
+  - **Three latest releases**: The three latest product release versions seen on each of your sources. This option can be helpful if the two latest release are not yet fully rolled out.
 
 ![Image shows the inspector Issues table ordered by issue volume and filtered by Web source.](/images/inspector/inspector-issues-filtering.png)
 
-<Callout type="info">We are actively working on adding more filter options, and you can expect more filters in the near future. If there’s anything specific you’d like to be able to filter by, please [let us know](/help/troubleshooting).</Callout>
+<Callout type="info">
+  We are actively working on adding more filter options, and you can expect more
+  filters in the near future. If there’s anything specific you’d like to be able
+  to filter by, please [let us know](/help/troubleshooting).
+</Callout>
+
+## Issue details
+
+Click any issue in the list to see more details. If you have multiple sources connected with Inspector, you can dig into which sources are impacted by the issue and uncover which sources are most severely affected.
+
+<br />
+<center>
+  <img
+    src="/docs/images/inspector/inspector-issue-details.png"
+    width="500px"
+    alt="Image shows example issue details for an unexpected property."
+  />
+</center>
+
+### Event info
+
+The event impacted by the issue, its volume and whether it is documented in your Avo tracking plan.
+
+- If the event is present in the tracking plan, you can navigate to it to review and/or update the event definition
+- If the event is missing from your tracking plan, you can add it to your tracking plan.
+
+### Property info
+
+The property impacted by the issue, and whether it is documented to be sent with the event in your tracking plan.
+
+- If the property is present on the event in your tracking plan, you can navigate to it to review and/or update.
+- If the property is present in your tracking plan but not set to be sent with your event, you can add it to the event definition.
+- If the property is missing from your tracking plan, you can add it to your tracking plan.
```

Comment:
We don't have a CTA button for that yet (but hopefully next week )

Reply:
Removed in 112baf24db841b0a5f27f1a62b829d2921e3f52f